ncd-calculator/src/components/FastaSearch.jsx

```diff
@@ -125,15 +125,15 @@ export const FastaSearch = () => {
     }
 
 
-    const performSearch = async (searchTerms, projectionOptions) => {
+    const performSearch = async (searchTerms, projectionOptions, apiKey) => {
```

Comment:
Please find another way to use the API key, I think we shouldn't have another parameter `apiKey` everywhere we try to fetch the GenBank sequences. 

It would be nice if you could keep the fetching functions intact, and you need to intercept the API key everywhere when it's available. Creating a state variable to manage the API key, and then enable/disable FASTA selector accordingly.

Reply:
i think separate parameter better.

Reply:
Yeah, let's keep the `apiKey` parameter.